README.md

```diff
@@ -40,7 +38,40 @@ and the requisite tables will be created.
 
 ## Usage
 
-TODO: Write usage instructions here
+Add your Pivotal Tracker API token to your secrets.yml:
+
+`tracker_api_token: <%= ENV["PIVOTAL_TRACKER_API_TOKEN"] %>`
+
+Create a new rake task called `import.rake`
```

Comment:
Hmm interesting, I'll look into that. That would be useful. My only concern is making sure the user can specify the interval